app/gui/view/graph-editor/src/component/node/growth_animation.rs

```diff
@@ -0,0 +1,99 @@
+//! Edited node growth/shrink animation implementation.
+//!
+//! When the user starts editing of the node - it smoothly growth in size to match the 1.0x zoom
+//! factor size. After editing is finished, the node smothly shrinks to its original size. This is
+//! implemented by using a separate `edited_node` camera that is moved in synchronization with
+//! `node_searcher` camera.
+
+use ensogl::prelude::*;
+
+use crate::GraphEditorModelWithNetwork;
+use crate::NodeId;
+use enso_frp as frp;
+use ensogl::animation;
+use ensogl::display::Scene;
+use ensogl::Animation;
+use ensogl::Easing;
+
+/// Initialize edited node growth/shrink animator. It would handle scene layer change for the edited
+/// node as well.
+pub fn initialize_edited_node_animator(
+    model: &GraphEditorModelWithNetwork,
+    frp: &crate::Frp,
+    scene: &Scene,
+) {
+    let network = &frp.network;
+    let out = &frp.output;
+    let searcher_cam = scene.layers.node_searcher.camera();
+    let edited_node_cam = scene.layers.edited_node.camera();
+    let main_cam = scene.layers.main.camera();
+
+    let growth_animation = Animation::new(network);
+    let animation_blending = Easing::new(network);
+
+    frp::extend! { network
+        let searcher_cam_frp = searcher_cam.frp();
+        let main_cam_frp = main_cam.frp();
+
+
+        // === Starting  node editing. ===
```

Comment:
double space + we are not using dots in section titles - in English, section titles are dotless sentences. Which is kind of stupid, but it is this way.

Reply:
Fixed!